anabot/runtime/installation/hub/software_selection.py

```diff
@@ -216,3 +223,43 @@ def addon_check(element, app_node, local_node):
     if action_result(element)[0] == False:
         return action_result(element)
     return addon_handler_manipulate(element, app_node, local_node, True)
+@handle_act('/kernel_options')
+def kernel_options_handler(element, app_node, local_node):
+    """ Functions correctly only in english as I didn't find a way to translate the full kernel options."""
+    chosen_kernel_option = get_attr(element, "value")
+    if chosen_kernel_option is None:
+        return ActionResultFail("No kernel options specified in the anabot recipe.")
+    try:
+        kernel_combo_box = getnode(local_node, "combo box",
+                                   tr(DEFAULT_KERNEL_OPTION, context="GUI|Software Selection|Kernel Options"))
+    except TimeoutError:
+        return NotFoundResult(f"Kernel Options combo box named {DEFAULT_KERNEL_OPTION}",
+                              where="Software Selection Spoke.")
+    # Open the combo box window by clicking on the combo box
+    kernel_combo_box.click()
+    try:
+        kernel_option_window = getnode(app_node, "window")
+    except TimeoutError:
+        return NotFoundResult("the opened 'Kernel Options' window", where="Software Selection Spoke.")
+    # The wanted kernel option node is specified by its full name, so we need to get it in order for anabot to find it
+    kernel_option_full_name = kernel_options.get(chosen_kernel_option, None)
```

Comment:
It might be worth checking that a value has been found using the supplied attribute value and return ActionResultFail() if not. OTOH the proper values are documented and the room for a mistake in a newly created recipe is quite small, so this isn't a hard requirement at all - it's up to your judgement.